GOVERNANCE.md

```diff
@@ -142,3 +142,22 @@ in additon to the ones listed for changing CPC governance:
 
 If consensus cannot be reached, a pull request may still be landed after a vote
 by the Voting CPC members to override outstanding objections.
+
+### Fast-Tracking PRs
+
+Special exception is made for pull requests seeking to make any of the following
+changes to this repository:
```

Comment:
I don’t think this apply to the CPC_CHARTER.

Reply:
I'd agree that specifically mentioning that this should exclude CHARTER changes makes sense.